src/utils/previewUtils.ts

```diff
@@ -27,7 +27,7 @@ export function getPreviewIfAny({ fileid, filename, previewUrl, hasPreview, davP
 		+ `&x=${Math.floor(screen.width * devicePixelRatio)}`
 		+ `&y=${Math.floor(screen.height * devicePixelRatio)}`
 		+ '&a=true'
-		+ (etag !== null ? `&etag=${etag.replace(/&quot;/g, '')}` : '')
+		+ (etag !== null ? `&etag=${String(etag).replace(/&quot;/g, '')}` : '')
```

Comment:
```suggestion
		+ (typeof etag === 'string' ? `&etag=${String(etag).replace(/&quot;/g, '')}` : '')
```

Reply:
Ok, given the issue, I think the original solution is more appropriate. Might simply be an issue of parsing the DAV response.

Reply:
Or maybe

```suggestion
		+ (etag ? `&etag=${String(etag).replace(/&quot;/g, '')}` : '')
```